### DISCUSSION TITLE
Moving a QSO logged to the wrong logbook or the wrong station location

### DISCUSSION THREAD

**Discussion by mwfoutch** [magicbug/Cloudlog#3078] [Q&A] [answered]
Have you ever logged a QSO to the wrong logbook and/or the wrong station location?

What's the easiest way to move a QSO that was logged in the wrong place?

Do I need to recreate the QSO in the correct logbook / station location and delete the original entry?

**Comment by phl0** [comment, marked as answer]
You can edit the QSO and change the station location:

![image](https://github.com/magicbug/Cloudlog/assets/7112907/fa4d686e-87e3-4ae2-93c3-ba39608a7ac6)

**Comment by magicbug** [comment]
@phl0 got there before me but what he said :p
